Explorer/Assets/DCL/Character/CharacterMotion/Systems/InterpolateCharacterSystem.cs

```diff
@@ -46,7 +47,18 @@ private void TeleportPlayer(in Entity entity, in CharacterController controller,
         }
 
         [Query]
-        [None(typeof(PlayerTeleportIntent))]
+        private void MovePlayer(in Entity entity,in CharacterController controller, ref CharacterPlatformComponent platformComponent, in PlayerMoveIntent moveIntent)
+        {
+            platformComponent.CurrentPlatform = null;
+
+            // Move the character inside the scene
+            controller.transform.position = moveIntent.Position;
+
+            World.Remove<PlayerMoveIntent>(entity);
+        }
```

Comment:
This is the same logic as `TeleportPlayer` method with `PlayerTeleportIntent` which is only used to reposition the character. We should merge both into something with a better name like `ForcePlayerReposition`

Reply:
You're completely right, at the end the `MovePlayer` query ended having the same logic as the `TeleportPlayer` one. I have removed `MovePlayer` and now both, move & teleport player features, use the same logic.

About the name, I think it's not bad the concept of "teleport" for both behaviors. At the end of the day both things do the same, teleport the player to any place. The only difference is if the player is moved to a coordinate out or inside the current scene. So, if you agree, lets keep the current `TeleportPlayer` name for the query.

Changes done here: https://github.com/decentraland/unity-explorer/pull/322/commits/078795933354b6000fb523606f1044152795ea32